toolset/databases/mysql/create.sql

```diff
@@ -14,8 +14,12 @@ CREATE TABLE  world (
 ENGINE=INNODB;
 CREATE USER 'benchmarkdbuser'@'%' IDENTIFIED WITH mysql_native_password BY 'benchmarkdbpass';
 CREATE USER 'benchmarkdbuser'@'localhost' IDENTIFIED WITH mysql_native_password BY 'benchmarkdbpass';
+CREATE USER 'benchmarkdbuser_sha2'@'%' IDENTIFIED WITH caching_sha2_password BY 'benchmarkdbpass';
```

Comment:
@jderusse If all users of mysql can benefit from this change without any code changes, than let's apply this to the `benchmarkdbuser` otherwise, I don't want to make this change for one framework.

Reply:
This is not about 1 framework, this is about all PHP frameworks using mysqlnd driver and the latstest version of PHP (7.4).

As far as I know, the authentication method is not configurable (nor in framework, nor in php.ini itself), the password have to use the plugin understood by the driver.
mysql_native_password for libmysql or mysqlnd with (PHP<7.4 and many other languages)
caching_sha2_password for mysqlnd (PHP>=7.4)

Reply:
@jderusse `mysqlnd` supports `mysql_native_password`

Please add 

```
default_authentication_plugin = mysql_native_password
```

to toolset/databases/mysql/my.cnf and remove the new user. That should work.

Reply:
much better. Thank you